bool automatic_stepping #True=enable automatic stepping
---

